Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM --platform=${BUILDPLATFORM} docker.io/library/golang:1.24-alpine AS builder
 
-ARG LDFLAGS
```

Comment:
Why was this removed?

Reply:
we don't really need it, it's already properly set in the Makefile and Dockerfile (hardcoded).